docs/faq.md

```diff
@@ -80,3 +81,28 @@ Currently, the main benefit of [signed releases](/checks.md#signed-releases) is
 However, there are already moves to make it even more relevant. For example, the OpenSSF is working on [implementing signature verification for NPM packages](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/) which would allow a consumer to automatically verify if the package they are downloading was generated through a reliable builder and if it is correctly signed.
 
 Signing releases already has some relevance and it will soon offer even more security benefits for both consumers and maintainers.
+
+### Branch-Protection: How to setup a 10/10 branch protection on GitHub?
+
+To get a 10/10 score for Branch-Protection check using a non-admin token, you should have the following settings for your branches:
+
+![GitHub's branch protection settings with the following options selected: "Require a pull request before merging", "Require approvals" with 1 approver, "Require review from Code Owners", "Require status checks to pass before merging", "Require branches to be up to date before merging", and have at least one Status Check chosen. All other options are unchecked.](/docs/design/images/branch-protection-settings-non-admin-token.png)
+
+When using an admin token, Scorecard can verify if a few other important settings are ensured:
```

Comment:
Can we pick a different example status check for these pictures?
I don't want to imply that someone needs to make `Scorecard analysis` as one of the required status checks before merging.

Even just changing it with `Inspect Elements` for the purpose of the screenshot is fine

Reply:
Sure!